source/detail/serialization/xlsx_producer.cpp

```diff
@@ -2180,7 +2180,8 @@ void xlsx_producer::write_worksheet(const relationship &rel)
                 return true;
             }
 
-            for (auto row = ws.lowest_row(); row <= ws.highest_row(); ++row)
+            auto highest = ws.highest_row();
```

Comment:
This is currently a linear search. Should really cache highest/lowest colun/row to make this a constant time operation